podcasts/End of Year/StoriesModel.swift

```diff
@@ -107,11 +107,11 @@ class StoriesModel: ObservableObject {
 
 private extension StoriesModel {
     func subscribeToNotifications() {
-        StoriesController.Notifications.allCases.forEach { controller in
+        StoriesController.Notifications.allCases.forEach { [weak self] controller in
```

Comment:
@emilylaguna FYI: this fixes the memory leak that was causing issues with the story progress indicator.